### DISCUSSION TITLE
How to get decent results with SDXL

### DISCUSSION THREAD

**Discussion by bkutasi** [blepping/comfyui_jankhidiffusion#23] [General]
Seems to be quite a tricky task, but HiDiffusion eclipsed every solution I found for 1-step generating high-res pictures.
In my experience the ca_start_time and end time makes things extremely messy, and somehow it ends up generating a lot of patterns (cant generate a person with a single color background, or a clean cityscape). 
Adding "trash" or busyness to the composition solves this or e.g.: prompting for an autumn forest. But I found out we can get rid of this by lowering the ca_, and I managed to generate some very good backgrounds with the following settings:
![image](https://github.com/blepping/comfyui_jankhidiffusion/assets/47147160/6b8e65f7-1290-4304-84ba-e89a5cf01856)
I mainly use 2048x2048, or fractions of it, for portraits I would go with 1536x2048. Should be generally compatible with every model, I mainly used DreamShaper lightning.
Interestingly, laying people will lead to the same type of body horror as SD3. It also has a problem with prompt following, so only the simple prompts and compositions will work.

I went over the original hidiffusion paper, and I'm pretty sure the examples were cherry-picked. I'm also sure my settings can be improved, so please try it and experiment with it. If someone is up for it, a combination can be created that generates all the possible settings and sorts out the results based on the aesthetic score.
Any ideas for improvements?

**Comment by LukeOvermind** [comment]
The above settings worked a charm for me up to until recently, which I guess is due to an update. Popping an old image into ComfyUI now gives me total different results and not for the better. 
![ComfyUI_temp_djyao_00022_](https://github.com/user-attachments/assets/bdd5667a-4238-4cd8-9a4e-1f79025e4b71)
![ComfyUI_temp_xqqcm_00027_ (1)](https://github.com/user-attachments/assets/6794a03b-27fc-4396-8a92-baef6e10dc4d)

In the attached images the freaky male figure on the hill is before the update and the other attachment is nothing as remotely close. 

Any suggestions as to what is causing this mayor change suddenly?

**Comment by blepping** [comment]
> Any suggestions as to what is causing this mayor change suddenly?

i haven't made any changes for about 2 weeks. without knowing what version you updated _from_, it's hard to help (actually, i don't even know if you're using the latest version currently either).

i've been fairly good about documenting user visible changes in the changelog: https://github.com/blepping/comfyui_jankhidiffusion/blob/main/changelog.md

you might not have generated what you want, but your images both look fine from a quality standpoint. if you updated other stuff at the same time, it's possible something just changed the seed, it doesn't look like a problem with jankhidiffusion parameters.
